docs/book/programs/classifiers.md

```diff
@@ -77,8 +77,8 @@ Here's how the code looks like:
 --8<-- "examples/tc-egress/tc-egress/src/main.rs"
 ```
 
-1. Loading the eBPF program.
-2. Attaching it to the given network interface.
+1. Get a reference to the map.
```

Comment:
Let's keep all the sentences grammatically consistent (using the same present tense) - before your change, all the pointy were in present continuous everywhere, but now you changed two points to present simple.

I'm fine with using any of those tenses, but if you want to stick to present simple, let's change the last point to:

```
3. Populate the map [...]
```

Reply:
Oh, yeah, that makes sense :+1:. Done